fiatconnect-api.md

```diff
@@ -2366,7 +2366,7 @@ If `keyType` is CPF, `key` MUST be a [CPF number](https://en.wikipedia.org/wiki/
 
 If `keyType` is `PHONE`, `key` MUST be an 11-digit [Brazilian mobile phone number](https://en.wikipedia.org/wiki/Telephone_numbers_in_Brazil). Area code MUST be included. The string MUST match the regex `/[0-9]{11}/` (a series of 11 digits, 0 through 9).
 
-Otherwise, if `keyType` is `RANDOM`, `key` MUST be a string of 32 alphanumeric characters. (In this last case, the key represents a random key generated by the user's bank.) 
+Otherwise, if `keyType` is `RANDOM`, `key` MUST be a string of 32 alphanumeric and dash characters. (In this last case, the key represents a random key generated by the user's bank.) 
```

Comment:
```suggestion
Otherwise, if `keyType` is `RANDOM`, `key` MUST be a string of 32 characters consisting only of numbers 0-9, [ISO basic Latin alphabetic](https://en.wikipedia.org/wiki/ISO_basic_Latin_alphabet) letters, or [hyphen-minus](https://en.wikipedia.org/wiki/Hyphen-minus). If `keyType` is `RANDOM`, `key` represents a random key generated by the user's bank.
```

Reply:
"dash" and hyphen are actually different, did you mean hyphen? https://en.wikipedia.org/wiki/Wikipedia:Hyphens_and_dashes

Reply:
also, please fix `fiatconnect-types` once this gets merged